src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -729,21 +802,102 @@ def select_units(self, unit_ids, format="memory", folder=None) -> "SortingAnalyz
             The unit ids to keep in the new SortingAnalyzer object
         folder : Path or None
             The new folder where selected waveforms are copied
-        format:
-        a
+        format : "auto" | "binary_folder" | "zarr"
+            The format of the folder.
+
         Returns
         -------
-        we :  SortingAnalyzer
+        analyzer :  SortingAnalyzer
             The newly create sorting_analyzer with the selected units
         """
         # TODO check that unit_ids are in same order otherwise many extension do handle it properly!!!!
-        return self._save_or_select(format=format, folder=folder, unit_ids=unit_ids)
+        return self._save_or_select_or_merge(format=format, folder=folder, unit_ids=unit_ids)
+
+    def merge_units(
+        self,
+        units_to_merge,
+        new_unit_ids=None,
+        censor_ms=None,
+        merging_mode="soft",
+        sparsity_overlap=0.75,
+        new_id_strategy="append",
+        format="memory",
+        folder=None,
+        verbose=False,
+        **job_kwargs,
+    ) -> "SortingAnalyzer":
+        """
+        This method is equivalent to `save_as()`but with a list of merges that have to be achieved.
+        Merges units by creating a new sorting analyzer object in a new folder with appropriate merges
+
+        Extensions are also updated to display the merged unit ids.
+
+        Parameters
+        ----------
+        units_to_merge : list/tuple of lists/tuples
```

Comment:
I am not super happy with this naming.
I know I did the same in the other PR but it is time to find a cristal clear naming

Reply:
lists_of_ids_to_merge?

Reply:
We finally `merge_unit_groups`